parm/config/config.fv3

```diff
@@ -156,8 +156,11 @@ case ${case_in} in
         ;;
 esac
 
-if [[ "${WRTTASK_PER_GROUP}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP=${npe_node_max} ; fi
-if [[ "${WRTTASK_PER_GROUP_GFS}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP_GFS=${npe_node_max} ; fi
+#While in theory this is likely the best way to go, in practice partiularly for C768 runs will fail
+#when using npe_node_max.  Additional write groups also did not get around errors. Should likely 
+#be revisted at another time.
+#if [[ "${WRTTASK_PER_GROUP}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP=${npe_node_max} ; fi
+#if [[ "${WRTTASK_PER_GROUP_GFS}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP_GFS=${npe_node_max} ; fi
```

Comment:
This should be tested in a short C384/C192 cycled experiment.

Reply:
This was tested, see logs here /scratch1/NCEPDEV/climate/Jessica.Meixner/hr1/updated/cyctest02 

There were some additional updates to the workflow from develop if I need to retest, let me know.

Reply:
Commenting out these lines appears necessary now based on my own cycled testing at C384C192L127. So I agree with this change for that resolution at least.

> This was tested, see logs here /scratch1/NCEPDEV/climate/Jessica.Meixner/hr1/updated/cyctest02

@JessicaMeixner-NOAA Similar to cyctest01, this cyctest02 experiment did not use the commented out lines in `config.fv3` that are included in this PR:

```
-bash-4.2$ pwd
/scratch1/NCEPDEV/climate/Jessica.Meixner/hr1/updated/cyctest02
-bash-4.2$ grep WRTTASK EXPDIR/cyctest02/config.fv3 | grep npe_node_max | grep if
if [[ "${WRTTASK_PER_GROUP}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP=${npe_node_max} ; fi
if [[ "${WRTTASK_PER_GROUP_GFS}" -gt "${npe_node_max}" ]]; then export WRTTASK_PER_GROUP_GFS=${npe_node_max} ; fi
```

Based on my own testing (mentioned in the comment above for the `nth_fv3` change), we should include the change in this PR to comment out the lines that check `WRTTASK` against `npe_node_max` but revert `nth_fv3` back to 1 for C384.

It would be good to know that commenting out these lines also works for other resolutions. May not need these lines at all now. I will spin up a C192C96L127 cycled ATM test on Hera to check.

Reply:
A cycled ATM-only C192C96L127 test on Hera with these `WRTTASK` lines commented out has run a gdasfcst and a few gdasefcs jobs without issue. Will let the test continue overnight to see how it is.

Reply:
Additional low res tests completed fine with regards to the `WRTTASK` change.